lib/matplotlib/testing/enforcedgc.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import gc
+import os
+from nose.plugins import Plugin
+
+
+class EnforcedGC(Plugin):
+    """This plugin adds option to call ``gc.collect`` after each test"""
+    enabled = False
+
+    def options(self, parser, env=os.environ):
+        env_opt = 'COLLECT_GC_AFTER_TEST'
```

Comment:
It was `CALL_GC_COLLECT_AFTER_TEST` and then I have reduced it.
I can't name things. T_T